app/controllers/v0/account_controls_controller.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module V0
+  class AccountControlsController < SignIn::ServiceAccountController
+    VALID_CSP_TYPES = %w[logingov idme dslogon mhv].freeze
+
+    def credential_index
+      raise Common::Exceptions::ParameterMissing, 'icn' if params[:icn].blank?
+
+      serialized_user_verifications = serialize_user_verifications(user_verifications: fetch_verifications_by_icn)
+      Rails.logger.info('[V0::AccountControlsController] credential_index',
+                        { icn: params[:icn], requested_by: @service_account_access_token.user_identifier })
+      render json: { data: serialized_user_verifications }
+    rescue ActiveRecord::RecordNotFound
+      render json: { error: 'UserAccount not found.' }, status: :not_found
+    rescue Common::Exceptions::ParameterMissing => e
```

Comment:
Can we get custom errors for all of these rather than the `common` ones? (Since this is part of the 'Sign In' component, and trying to continue using as little of the rest of the app as possible)